core/src/main/kotlin/org/evomaster/core/problem/rest/service/AbstractRestSampler.kt

```diff
@@ -252,7 +290,18 @@ abstract class AbstractRestSampler : HttpWsSampler<RestIndividual>() {
 
     private fun initForBlackBox() {
 
-        swagger = OpenApiAccess.getOpenAPIFromURL(configuration.bbSwaggerUrl)
+        // adding authentication from config should be moved here.
+        addAuthFromConfig()
+
+        //swagger = OpenApiAccess.getOpenAPIFromURL(configuration.bbSwaggerUrl)
```

Comment:
remove comment

Reply:
Hi Andrea,

Thanks for your comment. I have removed this comment.